cdap-docs/_common/common-build.sh

```diff
@@ -37,6 +38,7 @@ PROJECT="cdap"
 PROJECT_CAPS="CDAP"
 REFERENCE="reference-manual"
 SOURCE="source"
+SPHINX_MESSAGES="build/warnings.txt"
```

Comment:
As it is written to the build directory, it gets removed at the start of each run. See the function *clean*:
```
rm -rf ${SCRIPT_PATH}/${BUILD}
```
(I figured that was a good place to write it, as it always gets deleted at the start.)